src/views/progressionBarView.js

```diff
@@ -0,0 +1,20 @@
+export const createProgressionBar =(data)=>{
+    const element = document.createElement('div');
+    element.id='progression_bar'
+    const childElement = document.createElement('div')
+
+    element.appendChild(childElement);
+
+    const numberOfQuestion=data.questions.length;
+    let currentQuestion=data.currentQuestionIndex +1
+    let width = (currentQuestion * 100) / numberOfQuestion;
+    
+    if (currentQuestion > numberOfQuestion) {
```

Comment:
Should this check be done in this view? I think the questionPage should check if the progress bar is needed and only. call this function if the bar is needed